app/controllers/concerns/curation_concerns/single_use_links_controller_behavior.rb

```diff
@@ -44,7 +44,7 @@ def authorize_user!
       end
 
       def asset
-        @asset ||= ActiveFedora::Base.load_instance_from_solr(params[:id])
+        @asset ||= ActiveFedora::Base.find(params[:id])
```

Comment:
I think I put this in here to get show pages running on all Solr and no Fedora. Can we replace it with a ActiveFedora::SolrService call instead?